packages/oauth-adapters/src/oauthAuthenticationAdapter.ts

```diff
@@ -70,9 +73,17 @@ function isValid(oAuthToken: OAuthToken | undefined): oAuthToken is OAuthToken {
   return typeof oAuthToken !== 'undefined';
 }
 
-function isExpired(oAuthToken: OAuthToken) {
-  return (
-    typeof oAuthToken.expiry !== 'undefined' &&
-    oAuthToken.expiry < Date.now() / 1000
-  );
+function isExpired(oAuthToken: OAuthToken, clockSkew?: number) {
```

Comment:
You can export these function and use them in SDKs authMangers directly

Reply:
Exported isValid and isExpired to be used directly from the SDK